sway/commands.c

```diff
@@ -45,6 +45,15 @@ static sway_cmd cmd_bar;
 static sway_cmd cmd_bindcode;
 static sway_cmd cmd_bindsym;
 static sway_cmd cmd_border;
+// Border colors, maybe we should not do it the i3 way but
+// create our own config section color {} instead?
```

Comment:
This is a valid question, but should not be in the source, rather put it in the PR description so we can discuss it. :)

I agree that the syntax for defining colors is very strange considering the rest of the syntax used in i3.